README.markdown

```diff
@@ -468,7 +437,7 @@ Prefer the shortcut versions of type declarations over the full generics syntax.
 **Preferred:**
 ```swift
 var deviceModels: [String]
-var employees: [Int: String]
+var employees: [Int : String]
```

Comment:
@yfujiki I have no strong opinion, but I **think** I can see the logic.

I was in the `a: b` always camp. What made me consider this suggestion was looking into a couple of the standard library's dictionary declarations:

1. No whitespace to specify types: arguments, variables, properties, protocol conformance, etc. `let variable: Type` and `struct A: B` for example.
2. Whitespace to separate dictionary types: `[A : B]`. There is no other real parallel, but I see it as an extension of the array syntax: `[Type]`, where the `:` delineates each type (although the max is 2).

I give a neutral thumb here: :+1: :-1: .